.cspell.json

```diff
@@ -12,6 +12,6 @@
         "custom-words"
     ],
     "words": [],
-    "ignoreWords": [],
+    "ignoreWords": ["aureliusdev", "podname"],
```

Comment:
Instead of adding the words here, you can add them in `.cspell/custom-words.txt`. Can you please move them?